server/src/database/models/achievement.model.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  Table,
+  Column,
+  DataType,
+  PrimaryKey,
+  Model,
+  ForeignKey,
+  BelongsTo,
+  AllowNull
+} from 'sequelize-typescript';
+import { Player } from '.';
+
+// Define other table options
+const options = {
+  modelName: 'achievements',
+  updatedAt: false,
+  indexes: [
+    {
+      unique: true,
+      fields: ['playerId', 'type']
+    },
+    {
+      fields: ['playerId']
+    },
+    {
+      fields: ['type']
+    }
+  ]
+};
+
+@Table(options)
+export default class Achievement extends Model<Achievement> {
+  @PrimaryKey
+  @ForeignKey(() => Player)
+  @AllowNull(false)
+  @Column({ type: DataType.INTEGER, onDelete: 'CASCADE' })
+  playerId: number;
+
+  @BelongsTo(() => Player)
```

Comment:
Can you move these relationship variables to the end of the file?

It would be neat to have all model properties on the top, and all relationships on the bottom

Reply:
Funny enough I originally had all the relationship stuff at the bottom of the model. I only moved away because I saw many people doing this for possibly readability for a foreign key pairing.

Reply:
Moved all the relationship stuff in the models down to the bottom.